src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -1566,12 +1570,12 @@ private static Stream OpenStream(CommonMessageProvider messageProvider, string p
             return null;
         }
 
-        private static string ResolveRelativePath(CommonMessageProvider messageProvider, string path, string baseDirectory, DiagnosticBag diagnostics)
+        private static string? ResolveRelativePath(CommonMessageProvider messageProvider, string? path, string? baseDirectory, DiagnosticBag diagnostics)
```

Comment:
>string? [](start = 90, length = 7)

Is this change needed? It looks like the only caller is above which checks `path == null`.

Reply:
It's not strictly necessary. Was more trying to match the overall pattern of `path` and `baseDirectory` in the file. Will remove.